### PR TITLE
Switch to sphinx_symbiflow_theme

### LINKED ISSUE #209: Repository using wrong sphinx theme?
[It seems to be using `sphinx-material==0.0.32`](https://github.com/SymbiFlow/symbiflow-examples/blob/master/docs/requirements.txt#L2) rather than https://github.com/SymbiFlow/sphinx_symbiflow_theme ?

### PR DESCRIPTION
**Pull request by kgugala**
fixes #209

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -65,22 +65,10 @@
...
 
-html_theme = 'sphinx_material'
+html_theme = 'sphinx_symbiflow_theme'
+
 html_theme_options = {
-    'nav_title': project,
-    'color_primary': 'deep-purple',
-    'color_accent': 'purple',
-    'repo_name': "symbiflow/symbiflow-examples",
-    'repo_url': 'https://github.com/symbiflow/symbiflow-examples',
```
Comment (mithro) [inline docs/conf.py:82]: You probably want to keep this - https://github.com/google/skywater-pdk/pull/341/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R165

Comment (kgugala) [reply docs/conf.py:82]: this doesn't seem to be supported in the symbiflow theme. The build says:
```
WARNING: unsupported theme option 'repo_url' given
```

this combined with https://github.com/SymbiFlow/symbiflow-examples/blob/master/.readthedocs.yml#L26 fails on RTD.

I suppose we should now use `github_url` (according to https://sphinx-symbiflow-theme.readthedocs.io/en/latest/customization.html)

Comment (kgugala) [reply docs/conf.py:82]: I added `github_url`

**Comment by mithro** [comment]
It doesn't seemed to have worked? https://symbiflow-examples--210.org.readthedocs.build/en/210/ ?

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
-Sphinx==3.3.0
-sphinx-material==0.0.32
-sphinx-tabs==1.3.0
-sphinx-jinja==1.1.1
+sphinx
+docutils==0.17
+git+http://github.com/SymbiFlow/sphinx_materialdesign_theme.git@master#egg=sphinx_symbiflow_theme
```
Comment (mithro) [inline docs/requirements.txt:3]: This is the wrong theme. 
```suggestion
git+https://github.com/SymbiFlow/sphinx_symbiflow_theme.git#egg=sphinx-symbiflow-theme
```

**Comment by mithro** [comment]
Seems to be working now! - https://symbiflow-examples--210.org.readthedocs.build/en/210/index.html

**Pull request merged**